boards/arm/stm32f407g_disc1/doc/stm32f407g_disc1.rst

```diff
@@ -0,0 +1,202 @@
+.. _stm32f407g_disc1_board:
+
+ST STM32F407G-DISC1 Discovery board
+###################################
+
+Overview
+********
+
+The STM32F407G-DISC1 Discovery kit features an ARM Cortex-M4 based STM32F407VG MCU
+with a wide range of connectivity support and configurations Here are
+some highlights of the STM32F407G-DISC1 board:
+
+
+- STM32 microcontroller in LQFP100 package
+- Extension header for all LQFP100 I/Os for quick connection to prototyping board and easy probing
+- On-board ST-LINK/V2 debugger/programmer with SWD connector
+- Flexible board power supply:
+
+       - USB VBUS or external source(3.3V, 5V, 7 - 12V)
+       - Power management access point
+
+- Eight LEDs:
+
+       - USB communication (LD1)
+       - 3.3 V power on (LD2)
+       - Four user LEDs: orange (LD3), green (LD4), red (LD5), and blue (LD6)
+       - 2 USB OTG LEDs for VBUS (LD7) and over-current (LD8)
+
+- Two push-buttons: USER and RESET
+- USB OTG FS with micro-AB connector
+- LIS302DL or LIS3DSH ST MEMS 3-axis accelerometer
+- MP45DT02 ST-MEMS audio sensor omni-directional digital microphone
+- CS43L22 audio DAC with integrated class D speaker driver
+
+.. image:: img/en.stm32f4_discovery.jpg
+     :width: 350px
+     :align: center
+     :height: 550px
+     :alt: STM32F407G-DISC1
+
+More information about the board can be found at the `STM32F407G-DISC1 website`_.
+
+Hardware
+********
+
+STM32F407G-DISC1 Discovery kit provides the following hardware components:
+
+- STM32F407VGT6 in LQFP100 package
+- ARM |reg| 32-bit Cortex |reg| -M4 CPU with FPU
+- 168 MHz max CPU frequency
+- VDD from 1.8 V to 3.6 V
+- 1 MB Flash
+- 192+4 KB SRAM including 64-Kbyte of core coupled memory
+- GPIO with external interrupt capability
+- 3x12-bit ADC with 24 channels
+- 2x12-bit D/A converters
+- RTC
+- Advanced-control Timer
+- General Purpose Timers (17)
+- Watchdog Timers (2)
+- USART/UART (6)
+- I2C (3)
+- SPI (3)
+- SDIO
+- 2xCAN
+- USB 2.0 OTG FS with on-chip PHY
+- USB 2.0 OTG HS/FS with dedicated DMA, on-chip full-speed PHY and ULPI
+- 10/100 Ethernet MAC with dedicated DMA
+- 8- to 14-bit parallel camera
+- CRC calculation unit
+- True random number generator
+- DMA Controller
+
+More information about STM32F407VG can be found here:
+       - `STM32F407VG on www.st.com`_
```

Comment:
add a blank line before this bullet list, and don't indent the items:

    More information about STM32F407VG can be found here:

    - `STM32F407VG on www.st.com`_
    - `STM32F407 reference manual`_